sites/public/src/components/home/Home.module.scss

```diff
@@ -3,36 +3,44 @@
   .hero-container {
     background-color: var(--seeds-bg-color-surface-primary-inverse);
     padding-block: var(--seeds-s24);
+    background-image: url("/images/hero-main.jpg");
+    background-size: cover;
```

Comment:
issue: The design has the hero image not take up the full width with a border radius. I'm assuming that is supposed to be covered with this ticket, but let me know if there is a different ticket that is going to tackle that.
![image](https://github.com/user-attachments/assets/b2bf2a81-7781-4734-858e-399cbf857bc8)